app/init.php

```diff
@@ -175,6 +178,15 @@ function($value) {
 $register->set('cache', function () { // Register cache connection
     $redis = new Redis();
     $redis->pconnect(App::getEnv('_APP_REDIS_HOST', ''), App::getEnv('_APP_REDIS_PORT', ''));
+    $auth = App::getEnv('_APP_REDIS_AUTH',null);
```

Comment:
Use 2 env vars, if one of them is not empty use authentication.

Reply:
Updated!